index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Apple Clone</title>
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.7/css/all.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+</head>
+
+<body>
+    <main>
+        <div class="header-section">
+            <nav class="gradient justify-center">
+                <ul class="grid cursor">
+                    <li><a href="#"><i class="fab fa-apple"></i></a></li>
```

Comment:
[Optional] We can use an apple image for the logo or can increase the size of the font awesome icon.